eng/Version.Details.xml

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Accidental newline?

Reply:
odd.   same thing happened on Versions.props when I was rebasing and fixing a merge.  I noticed that before submitting but missed this.

Reply:
My guess would be it has to do with line endings or BOM due to how Maestro++/Darc updates it.